src/superface/mock/timers.ts

```diff
@@ -0,0 +1,48 @@
+import { ITimeout, ITimers } from '@superfaceai/one-sdk';
+
+export class MockTimeout implements ITimeout {
+  constructor(public value: number) {}
+}
+
+export class MockTimers implements ITimers {
```

Comment:
Why didn't you just reused mocks from SDK?

Reply:
Because it's not exported from sdk as far as I know 😄